index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>First Quiz</title>
+  <link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+<!-- articles created for each product to be displayed, includes sections with info about product -->
+  <article>
+    <!-- title of prodect -->
```

Comment:
These kind of comments don't really add context since the next line is `class="productTitle"`